src/pages/home/sidebar/SidebarScreen/FABActionsPopover.js

```diff
@@ -0,0 +1,180 @@
+                        },
+                        ...(Permissions.canUsePolicyRooms(this.props.betas) && workspaces.length ? [
+                            {
+                                icon: Expensicons.Hashtag,
+                                text: this.props.translate('sidebarScreen.newRoom'),
+                                onSelected: () => Navigation.navigate(ROUTES.WORKSPACE_NEW_ROOM),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOUSend(this.props.betas) ? [
+                            {
+                                icon: Expensicons.Send,
+                                text: this.props.translate('iou.sendMoney'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_SEND),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOU(this.props.betas) ? [
+                            {
+                                icon: Expensicons.MoneyCircle,
+                                text: this.props.translate('iou.requestMoney'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_REQUEST),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOU(this.props.betas) ? [
+                            {
+                                icon: Expensicons.Receipt,
+                                text: this.props.translate('iou.splitBill'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_BILL),
+                            },
+                        ] : []),
+                        ...(!Policy.hasActiveFreePolicy(this.props.allPolicies) ? [
+                            {
+                                icon: Expensicons.NewWorkspace,
+                                iconWidth: 46,
+                                iconHeight: 40,
+                                text: this.props.translate('workspace.new.newWorkspace'),
+                                description: this.props.translate('workspace.new.getTheExpensifyCardAndMore'),
+                                onSelected: () => Policy.createWorkspace(),
+                            },
+                        ] : []),
+                    ]}
+                />
+                <FloatingActionButton
+                    accessibilityLabel={this.props.translate('sidebarScreen.fabNewChat')}
+                    accessibilityRole="button"
+                    isActive={this.state.isCreateMenuActive}
+                    onPress={this.showCreateMenu}
+                />
+            </View>
+        );
+    }
+}
+
+FABActionsPopover.propTypes = propTypes;
+FABActionsPopover.defaultProps = defaultProps;
+
+export default compose(
+    withLocalize,
+    withNavigation,
+    withWindowDimensions,
+    withOnyx({
+        allPolicies: {
+            key: ONYXKEYS.COLLECTION.POLICY,
+        },
```

Comment:
Please use a selector here (you can see how they are used in SidebarLinks). I believe the only properties you need are `type` and `role`.

Reply:
oh awesome that we have selectors now, updated the code!